_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -85641,6 +85634,13 @@
 	icon_state = "4-8"
 	},
 /obj/effect/spawner/random_spawners/dirt_often,
+/obj/structure/sign/securearea{
```

Comment:
Swap it for /obj/structure/sign/vacuum/external since you are touching it.

Reply:
Done.